include/vinecopulib/vinecop/fit_controls.hpp

```diff
@@ -27,6 +27,7 @@ class FitControlsVinecop : public FitControlsBicop
                        std::string tree_criterion = "tau",
                        double threshold = 0.0,
                        std::string selection_criterion = "bic",
+                       double psi0 = 0.9,
```

Comment:
Haha, so you think that this one will be used more often than select_truncation_level/threshold ?

Reply:
no, I didn't really think about the positioning. should we put it before `num_threads`?

Reply:
don't know actually, no strong opinion on this, and actually wondering the same about the threshold

Reply:
honestly, I don't think many people will use the constructor with more than the first 1-3 arguments. I always use `.set_xxx()` as soon there are more than 2.